src/wireguard.rs

```diff
@@ -0,0 +1,305 @@
+//! Wireguard
+/**
+
+no permissions / unknown user : 
+
+     Authenticate                    params: PIN/pass         - Unlocks the device to use it. 
```

Comment:
Just FYI, I am planning to add a "global PIN" to Trussed, so apps don't necessarily have to do their own PIN handling. Depending on the app, they of course may choose to do so all the same.